servicedesk-scripts/reboot_instance.py

```
@@ -0,0 +1,57 @@
+import boto3
+import time
+import sys
+import argparse
+
+
+region = 'us-east-1'
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--elbname', required=True, type=str)
+args = parser.parse_args()
+elbname = args.elbname
+
+def main():
+    session = boto3.Session(profile_name='cf-invalidation')
+
+    elb = session.client('elb', region_name=region)
+
+    instance = elb.describe_load_balancers(LoadBalancerNames=[elbname,])['LoadBalancerDescriptions'][0]['Instances'][0]['InstanceId']
```

Keep in mind this is dependent on only one instance per load balancer, you could throw an error on the script if the number of instances is more than one.

---

Also in the description, clarify that this is a caveat.